### PR TITLE
Fixes MoMMIs dropping items while inside vents

### PR DESCRIPTION
**Pull request by 9600bauds**
Slays fun

### CODE REVIEW

**Comment by ComicIronic** [comment]
What stops MoMMIs dropping stuff in lockers? I feel like this is in need of a systems change and not a special exemption.

**Comment by Exxion** [comment]
If you drop something in a locker, wouldn't it just get dropped into the locker?

**Comment by AlphaBuffalo** [comment]
If a mommi drops say the nuke disk in the pipes, would you have to remove said pipe to get it?

**Comment by ComicIronic** [comment]
If they drop something in a locker, it will still end up on the outside turf, not inside the locker.

**Comment by Exxion** [comment]
Well that's fucking stupid then.
Why doesn't it work like
I don't know
Any other mob dropping something?

**Comment by 9600bauds** [comment]
I'll look at this later

**Comment by ComicIronic** [comment]
I don't know, can a contortionist human bring the nuke disk into a pipe and drop it?
I mean, you could always get the disk out by unwrenching.

**Comment by 9600bauds** [comment]
It doesn't work like any other mob dropping something because silicon inventory is shit and MoMMI aggravated this by copypasting every inventory proc back in and now oh woe is me all of those are outdated

**Comment by 9600bauds** [comment]
Now MoMMIs can't play around with their modules while ventcrawling but then again no other mob can do that with their own inventory even while inside a locker so eh

**Comment by clusterfack** [comment]
I went to refactor mommi code once but there is literally about 50% unnecessary code in their inventory code. I'm not joking you can rip out 50% of mommi inventory code and it will work EXACTLY the same as it does now. Also they cant deselect their hand anymore which is annoying because it let you easily reach long range shit without putting away your module every time.

**Comment by 9600bauds** [comment]
I fully believe you

File: code/modules/mob/living/silicon/mommi/inventory.dm [hunk 1]
```diff
@@ -115,15 +115,13 @@
 		contents -= tool_state
 		var/obj/item/TS = tool_state
 		if(!Target)
-			Target = get_turf(src)
+			Target = src.loc
```
Comment (PJB3005) [inline code/modules/mob/living/silicon/mommi/inventory.dm]: `src.`

**Comment by PJB3005** [comment]
Until somebody wants to fix MoMMI inventory code this will have to do.

**Pull request merged**